bokeh/models/plots.py

```diff
@@ -72,7 +74,13 @@
 from .axes import Axis
 from .glyphs import Glyph
 from .grids import Grid
-from .layouts import LayoutDOM
+from .layouts import (
+    ColSizing,
+    IntOrString,
+    LayoutDOM,
+    QuickTrackSizing,
```

Comment:
Is there some reference or origin for "Quick Track" ? I couldn't find anything from a quick GIS and I've never heard of it

Reply:
This is an obscure and barely used part of grid layout, where one can configure layout by setting up sizing of rows and columns, and inferring sizing from children. For now `GridPlot` just reimplements what `GridBox` does. Going forward, I will need to think if this is worthwhile keeping, or whether I should simplify the layout. That will happen when I will finally get to entangling grids with subcoordinates.